### PR TITLE
[GNA] add MultipleActivationsTest

### PR DESCRIPTION
**Pull request by a-noskov**
### Details:
 - MultipleActivationsTest from dldt tests closed (graph attached) -> MatMulActEltwTest
image

### Tickets:
 - 52043

### CODE REVIEW

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/multiple_activations.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2021 Intel Corporation
```
Comment (elilobanova) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/multiple_activations.cpp:1]: 2018 is extra in the new file
the same is for other added files

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/multiple_activations.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+namespace SubgraphTestsDefinitions {
+namespace {
+const std::vector<size_t> input_sizes = {
+    25,
+    30,
+    50
+};
+
+// const std::vector<size_t> output_fc = {
```
Comment (elilobanova) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/multiple_activations.cpp:18]: please, delete the commented code

File: inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/multiple_activations.hpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+namespace SubgraphTestsDefinitions {
+
+typedef std::tuple<
+        std::size_t,                        // Input Size
+        InferenceEngine::Precision,         // Network Precision
+        std::string,                        // Target Device
+//        std::size_t,                        // Output Size for FC layer
```
Comment (elilobanova) [inline inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/multiple_activations.hpp:22]: please, delete the commented code

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_activations.cpp [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    InferenceEngine::Precision netPrecision;
+    size_t inputSize;
+    std::map<std::string, std::string> config;
+    std::tie(inputSize, netPrecision, targetDevice, config) = this->GetParam();
+    configuration.insert(config.begin(), config.end());
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+    auto input = ngraph::builder::makeParams(ngPrc, { {1, inputSize} });
+
+    // auto relu = ngraph::builder::makeActivation(input[0], ngPrc, ngraph::helpers::ActivationTypes::Relu);
```
Comment (elilobanova) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_activations.cpp:35]: please, delete the commented code

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_activations.cpp [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+    // auto concat_1 = ngraph::builder::makeConcat({concat_const_1, relu}, 1);
+    // auto concat_2 = ngraph::builder::makeConcat({concat_const_2, relu}, 1);
+
+    std::vector<size_t> outFormShapes = {1,  2 * inputSize};
+    std::vector<float> fc1_weights = CommonTestUtils::generate_float_numbers(inputSize * outFormShapes[1], -0.5f, 0.5f);
+    auto fc1 = ngraph::builder::makeFullyConnected(
+        input[0], ngPrc, outFormShapes[1], false, {}, fc1_weights);
+
+    auto tanh2 = ngraph::builder::makeActivation(fc1, ngPrc, ngraph::helpers::ActivationTypes::Tanh);
```
Comment (elilobanova) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_activations.cpp:50]: why is this test called multiple activations? As soon as I see only one activation (tanh_2) is created here

File: inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/multiple_activations.cpp [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+//     100,
+//     200
+// };
+
+const std::vector<InferenceEngine::Precision> net_precisions = {
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP16
+};
+
+std::map<std::string, std::string> additional_config = {};
```
Comment (elilobanova) [inline inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/multiple_activations.cpp:29]: why is additional config empty? you could add GNA mode here (GNA_SW_FP32 and GNA_SW_EXACT)

**Review by elilobanova — APPROVED** [review]

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/matmul_multiple_outputs.cpp [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+    auto matmul_1 = std::make_shared<ngraph::op::MatMul>(params[0], constant_1, false, true);
+
+    auto tanh2 = ngraph::builder::makeActivation(matmul_1, ngPrc, ngraph::helpers::ActivationTypes::Tanh);
+
+    auto eltw = ngraph::builder::makeEltwise(matmul_1, tanh2, ngraph::helpers::EltwiseTypes::ADD);
+
+    function = std::make_shared<ngraph::Function>(eltw, params, "Muliple_Activations");
+}
+} // namespace SubgraphTestsDefinitions
```
Comment (nkogteva) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/matmul_multiple_outputs.cpp:47]: BTW, if this test indent to check MatMul with multiple outputs only, why we need all subgraph here? Maybe move this check to single layer tests?

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/matmul_act_eltw.cpp [hunk 8]
```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2021 Intel Corporation
```
Comment (mryzhov) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/matmul_act_eltw.cpp:1]: Are you testing different eltwise operations? I see only add in the test.
Let's rename the test to mamul_tanh_add

Comment (mryzhov) [reply inference-engine/tests/functional/shared_test_classes/src/subgraph/matmul_act_eltw.cpp:1]: and what activation type was used in the legacy test?

Comment (a-noskov) [reply inference-engine/tests/functional/shared_test_classes/src/subgraph/matmul_act_eltw.cpp:1]: > and what activation type was used in the legacy test?

Also Tanh, the graph attached in the description

**Pull request merged**